resources/js/package.js

```diff
@@ -59,96 +91,80 @@ function init() {
+                        }
+                    },
+                    setSearchParams(url) {
+                        window.history.pushState(window.history.state, '', new URL(url))
+                    },
+                    toggleScroll(bool = null) {
+                        if (bool === null) {
+                            this.scrollLock = !this.scrollLock
+                        } else {
+                            this.scrollLock = bool
+                        }
+                    },
+                    resizedPath(imagePath, size, store = null) {
+                        if (!store) {
+                            store = window.config.store
+                        }
 
-    const lastStoreCode = useLocalStorage('last_store_code', window.config.store_code)
-    if (lastStoreCode.value !== window.config.store_code) {
-        clearAttributes()
-        clearSwatches()
-        lastStoreCode.value = window.config.store_code
-    }
+                        let url = new URL(imagePath)
+                        url = url.pathname.replace('/media', '')
 
-    if (window.debug) {
-        window.app.$on('notification-message', function (message, type, params, link) {
-            switch (type) {
-                case 'error':
-                    console.error(...arguments)
-                    break
-                case 'warning':
-                    console.warn(...arguments)
-                    break
-                case 'success':
-                case 'info':
-                default:
-                    console.log(...arguments)
-            }
-        })
-    }
+                        return `/storage/${store}/resizes/${size}/magento${url}`
+                    },
+                },
+                computed: {
+                    // Wrap the local storage in getter and setter functions so you do not have to interact using .value
+                    guestEmail: wrapValue(
+                        useLocalStorage('email', window.debug ? 'wayne@enterprises.com' : '', { serializer: StorageSerializers.string }),
+                    ),
 
-    const event = new CustomEvent('vue:loaded', { detail: { vue: window.app } })
-    document.dispatchEvent(event)
+                    loggedIn() {
+                        return this.user?.is_logged_in
+                    },
+
+                    hasCart() {
+                        return this.cart?.id && this.cart.items.length
+                    },
+                },
+                watch: {
+                    loadingCount: function (count) {
+                        window.app.$data.loading = count > 0
+                    },
+                },
+            })
+
+            setTimeout(() => {
+                const event = new CustomEvent('vue:loaded', { detail: { vue: window.app } })
+                document.dispatchEvent(event)
+            })
+        }
+    )
 }
+
 document.addEventListener('turbo:load', init)
+setTimeout(init)
```

Comment:
This specific order of setTimeout and [requestAnimationFrame](https://github.com/rapidez/core/pull/646/files#diff-251b99eac1abebedfe294a2334032b29b5f843d0b5d0834c32c055635f79c935R94) ensures LCP is not impacted.
And ensures small task time for the `turbo:load`/evaluate script, and `vue:loaded` events by scheduling them all in their own task.

Using a setTimeout for both, a requestAnimationFrame for both, or executing this immediately and booting Vue in a requestAnimationFrame all has impact on the LCP